File: hw/cañarte/u2/HW18ZooInPython/ec_edu_espe_zoo_model/Bird.py
import sys
import os

sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
from Animal import Animal

# Saray Cañarte, codecreators, DCCO - ESPE

class Bird(Animal):
    def __init__(self, id, description, date_onborn, gender, number_of_bones, cage):
        super().__init__(id, description, date_onborn, gender, number_of_bones, cage)

File: ws/ccedeno/u2/WS24Zoo/Zoo/src/main/java/ec/edu/espe/zoo/model/Bird.java
package ec.edu.espe.zoo.model;

import java.util.Date;

/**
 *
 * @author Andres Cedeno, codecreators, DCCO-ESPE
 */
public abstract class Bird extends Animal {

    public Bird(int id, String description, Date dateOnborn, char gender, int numberOfBones, Cage cage) {
        super(id, description, dateOnborn, gender, numberOfBones, cage);
    }

}
